app/src/main/java/org/mozilla/focus/home/NewSettingsFragment.kt

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package org.mozilla.focus.fragment
+package org.mozilla.focus.home
```

Comment:
NewSettings isn't a part of home – it can be accessed from the browserOverlay too. I think it should be at the top-level.